specification/ai/data-plane/Face/preview/v1.1-preview.1/Face.json

```diff
@@ -0,0 +1,7948 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure AI Face API",
+    "version": "v1.1-preview.1",
+    "x-typespec-generated": [
+      {
+        "emitter": "@azure-tools/typespec-autorest"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "name": "endpoint",
+        "in": "path",
+        "description": "Supported Cognitive Services endpoints (protocol and hostname, for example:\nhttps://<resource-name>.cognitiveservices.azure.com).",
+        "required": true,
+        "type": "string"
+      }
+    ]
+  },
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "ApiKeyAuth": []
+    },
+    {
+      "OAuth2Auth": [
+        "https://cognitiveservices.azure.com/.default"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "ApiKeyAuth": {
```

Comment:
Please include a description in each of these securityDefinition for the generated REST API reference docs. 

See the [TypeSpec docs] for an example of using the @doc decorator to specify a description for the securityDefinition. 

[TypeSpec docs]: https://azure.github.io/typespec-azure/docs/reference/azure-style-guide#examples

Reply:
Updated.